src/main/java/spoon/reflect/declaration/CtTypeInformation.java

```diff
@@ -60,6 +60,11 @@
 	boolean isAnonymous();
 
 	/**
+	 * Return {@code true} if the referenced type is declared in a executable.
+	 */
+	boolean isEnclosingType();
```

Comment:
I love this one: `isLocalOrAnonymousClass` (as in java.lang.Class)

But it should be `isLocal`. We already have `isAnonymous`.